pinry/static/js/bookmarklet.js

```diff
@@ -1,15 +1,13 @@
 /**
- * Bookmarklet for Pinry
- * Descrip: This is trying to be as standalone a script as possible hence
- *          why it has built in helpers and such when the rest of the
- *          scripts make use of helpers.js. In the future i want to remove
- *          all dependencies on jQuery.
- * Authors: Pinry Contributors
- * Updated: Mar 4th, 2013
- * Require: None (dynamically loads jQuery if needed)
- */
-
-
+* Bookmarklet for Pinry
+* Descrip: This is trying to be as standalone a script as possible hence
+* why it has built in helpers and such when the rest of the
+* scripts make use of helpers.js. In the future i want to remove
+* all dependencies on jQuery.
+* Authors: Pinry Contributors
+* Updated: Mar 4th, 2013
+* Require: None (dynamically loads jQuery if needed)
+*/
```

Comment:
The formatting looked better before. And like in manage.py: This is unrelated to the vimeo integration.